src/components/include/utils/threads/thread.h

```diff
@@ -172,7 +173,11 @@ class Thread {
    */
   void stop();
 
-  void join();
+  /**
+   * @brief Starts join procedure.
+   * @param[in] force_stop If true stop() is called first otherwise is skipped.
+   */
```

Comment:
@pestOO, fixed in [ca04ae0](https://github.com/smartdevicelink/sdl_core/pull/707/commits/ca04ae01ea2e23d26e87ceb337e38dbe7bb04f1a) and [6a6fa50](https://github.com/smartdevicelink/sdl_core/pull/707/commits/6a6fa50f4c909cb87b2be555b462c1ae7a660cb1) (Qt/Win)